### PR TITLE
Add portofolio and education page

### PR DESCRIPTION
**Pull request by NARCISA-LUCIA**

### CODE REVIEW

File: src/app/education/education.component.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  border: 1px solid #dedede;
+  border-color: rgba(NaN, 222, 222, 0.5);
+  width: 80%;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.head-container {
+  display: flex;
+  width: 980px;
```
Comment (sabaudragos) [inline src/app/education/education.component.css]: The image and the text don't behave well if the size of the browser window is small. It breaks the container. Could you double check on your end? Removing  `width: 980px;` might fix it

File: src/app/education/education-card/education-card.component.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
+::ng-deep .mat-vertical-stepper-content {
+    visibility: visible !important;
+    height: 100% !important;
+  }
+  
+  .step-card{
```
Comment (sabaudragos) [inline src/app/education/education-card/education-card.component.css]: Could you pls apply the code style to this file, the code seems to be missaligned?

File: src/app/education/education.component.css [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+}
+
+.text1 {
+  color: #919191;
+  margin-bottom: 20px;
+  line-height: 1.6;
+  font-size: 10px;
+}
+
+.text {
```
Comment (sabaudragos) [inline src/app/education/education.component.css]: Could you change the class name to something more meaningfull? 
e.g. address-paragraph maybe?

File: src/app/education/education.component.css [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  margin-top: 30px;
+  margin-bottom: 50px;
+}
+
+img {
+  width: 300px;
+  height: 350px;
+}
+
+.text1 {
```
Comment (sabaudragos) [inline src/app/education/education.component.css]: Could you change the name pls to something more descriptive? I helps to better maintain the code.
e.g. header-paragraph maybe?

File: src/app/education/education.component.css [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+.expertise-box-left,
+.expertise-box-right {
+  margin-right: 10px;
+  width: 400px;
+  row-gap: 30px;
+  display: flex;
+  flex-direction: column;
+}
+
+.expertise-box-left-1,
```
Comment (sabaudragos) [inline src/app/education/education.component.css]: `expertise-box-left-inner` would work better than `expertise-box-left-1`
same for `expertise-box-right-2` which could be replaced by `expertise-box-right-inner`

**Review by sabaudragos — CHANGES_REQUESTED** [review]

**Comment by sabaudragos** [comment]
I've left a few minor comments. The rest of the changes look good!

File: src/app/education/education.component.css (hunk 1 above)
Comment (NARCISA-LUCIA) [reply src/app/education/education.component.css]: Done

File: src/app/education/education-card/education-card.component.css (hunk 2 above)
Comment (NARCISA-LUCIA) [reply src/app/education/education-card/education-card.component.css]: Done

File: src/app/education/education.component.css (hunk 3 above)
Comment (NARCISA-LUCIA) [reply src/app/education/education.component.css]: Done

File: src/app/education/education.component.css (hunk 4 above)
Comment (NARCISA-LUCIA) [reply src/app/education/education.component.css]: Done

File: src/app/education/education.component.css (hunk 5 above)
Comment (NARCISA-LUCIA) [reply src/app/education/education.component.css]: Perfect, thanks, done

**Review by sabaudragos — APPROVED** [review]

**Pull request merged**